src/python/grpcio/grpc/_cython/_cygrpc/aio/channel.pyx.pxi

```diff
@@ -28,11 +38,12 @@ cdef class AioChannel:
 
     def call(self,
              bytes method,
-             object deadline):
+             object deadline,
+             CallCredentials credentials):
         """Assembles a Cython Call object.
 
         Returns:
           The _AioCall object.
         """
-        cdef _AioCall call = _AioCall(self, deadline, method)
+        cdef _AioCall call = _AioCall(self, deadline, method, credentials)
```

Comment:
`CallCredentials` are only valid for secure channels. We should raise an exception if users are trying to pass in a `CallCredentials` to an insecure channel.

Reply:
Ping.